### ISSUE TITLE
icalBuddy 1.8.10 fails to build on Yosemite

### ISSUE THREAD

**Issue by chdiza** [ali-rantakari/icalBuddy#7]
Trying to build the tarball, by cd'ing there and typing "make" on Yosemite with Xcode 6.1 results in the following:

```
---- Compiling main app:
======================================
clang  -O3 -Wall -Wextra -Wno-unused-parameter -Werror -std=c99 -force_cpusubtype_ALL -mmacosx-version-min=10.5 -arch i386  -framework Cocoa -framework CalendarStore -framework AppKit -framework AddressBook -o icalBuddy icalBuddy.m icalBuddy[ABCDEFGHIJKLMNOPQRSTUVWXYZ]*.m ANSIEscapeHelper.m HG*.m IcalBuddy*.m *+HGAdditions.m
icalBuddyArgs.m:163:16: error: incompatible pointer types assigning to
      'NSMutableDictionary *' from 'NSDictionary *'
      [-Werror,-Wincompatible-pointer-types]
    configDict = [NSDictionary dictionaryWithContentsOfFile:filePath];
               ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [icalBuddy] Error 1
```

**Comment by LittleOrangeC** [comment]
I was able to reproduce this error.  

With OS X 10.10.1 and Xcode 6.1.1 I was able to successfully build version 1.8.8, but not 1.8.10.

**Comment by edasque** [comment]
I could install icalbuddy on Yosemite 10.10 with:

```
brew install ical-buddy
```

I understand that's not building it but if all you need is 1.8.10 installed, that'll work.

**Comment by LittleOrangeC** [comment]
I agree 1.8.8 builds, installs and works just fine. I was just confirming the build bug @chdiza observed with 1.8.10.

**Comment by ali-rantakari** [comment]
Thanks. Fixed in 6b27b1e
